src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseCrudStretchTests.ps1

```diff
@@ -0,0 +1,177 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+	.SYNOPSIS
+	Tests creating a stretch database on V12.0 server
+#>
+function Test-CreateStretchDatabase
+{
+	Test-CreateDatabaseInternal "12.0" "Southeast Asia"
```

Comment:
We should not have hard-coded locations, as this prevents tests from running in AzureChina, AzureStack, etc.